website/.env

```diff
@@ -0,0 +1 @@
+VITE_NFT_CONTRACT_ADDRESS="0x005E2c2a327cEBE03409221cb37Abf4911AFF791"
```

Comment:
Could we get this from the package to set it in a single place instead of multiple `.env` files?

Reply:
ok, so I unified it here [497ff90](https://github.com/hemilabs/tge-mint-page/pull/5/commits/497ff9008ea943522e9473eb7530b75cddfe95ff)

I had to hardcode it because how to access env variables would depend on the project using it (for example, vite does not uses `process.env` directly).

However, It may not be a big issue, as the address adds extra methods on top of the ERC721, so it's not a generic nft minter (I mean, it is, but some of the validations are custom)